### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HeinUision**
Updated text to represent current status of Falcon

### Please check if the PR fulfills these requirements ###

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**

**Other information:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,54 @@
-# DEITY Falcon - build back-end agnostic headless PWA websites
+# DEITY Falcon - The most sophisticated front-end platform
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/b8b60963bac941f081de64671ae8b7fd)](https://app.codacy.com/app/Deity/falcon?utm_source=github.com&utm_medium=referral&utm_content=deity-io/falcon&utm_campaign=Badge_Grade_Settings)
-[![Build Status](https://travis-ci.org/deity-io/falcon.svg?branch=master)](https://travis-ci.org/deity-io/falcon)
+[![Build Status](https://gitlab.com/deity-io/falcon/badges/master/pipeline.svg)](https://gitlab.com/deity-io/falcon/commits/master)
```
Comment (cane-lagerwaard) [inline README.md]: Why this change?

Comment (HeinUision) [reply README.md]: Ah, that's something that came from the Master branch. You can skip that change.

Comment (cane-lagerwaard) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -8,30 +8,47 @@
...
 - Service worker to provide application caching and PWA features such as offline capabilities and add to homescreen
 - Falcon's architecture allows it to be very modular, lightweight and scalable
+- **Commercial license** - A BigCommerce extension to quickly build a feature-rich PWA store
+- **Commercial license** - An Intershop extension to quickly build a feature-rich PWA store
+- **Commercial license** - An Algolia Search extension for AI powered search, faceted navigation and catalog presentation
+- **Commercial license** - A payment integration layer to easily connect any payment service provider, including Stripe, Adyen and Mollie
+- **Commercial license** - A hosted environment for optimal performance
 
-These are just a couple of advantages and use-cases, feel free to ask anything in the official slack channel [slack.deity.io](https://slack.deity.io)
+These are just a couple of advantages and features, feel free to contact us for more information at [deity.io/contact](deity.io/contact) or join our official slack channel [slack.deity.io](slack.deity.io).
```
Comment (Borales) [inline README.md]:
```suggestion
These are just a couple of advantages and features, feel free to contact us for more information at [deity.io/contact](https://deity.io/contact) or join our official slack channel [slack.deity.io](slack.deity.io).

```
